DataBase.py

```diff
@@ -0,0 +1,52 @@
+from peewee import *
+
+
+#DataBase connection
+conn = SqliteDatabase('data.db')
+
+class BaseModel(Model):
+    class Meta:
+        database = conn 
+
+# Определяем модель пользователя
+class User(BaseModel):
+    user_id = AutoField(column_name='userid')
+    username = TextField(column_name='username')
+    age = TextField(column_name='age')
+    sphere = TextField(column_name='sphere')
+    date = TextField(column_name='lastnewsupdate')
+
+
+    class Meta:
+        table_name = 'users'
+
+cursor = conn.cursor()
+
+
+
+def create_user(user_username, user_age, preferences):
+    User.create(username = user_username, age = user_age, sphere = preferences)
```

Comment:
When you have a keyword/parameter, there should be no spaces before and after `=` (PEP8)
PyCharm or sth like this may help you with following the standards.

So, here it'll be:
`    User.create(username=user_username, age=user_age, sphere=preferences)`